chatkit-core/src/main/kotlin/com/pusher/chatkit/presence/api/PresenceApiType.kt

```diff
@@ -0,0 +1,7 @@
+package com.pusher.chatkit.presence.api
+
+internal sealed class PresenceApiType {
```

Comment:
that's not ApiType as it's not used by Gson. we are parsing effectively into our internal type, which indeed models additional "unknown"state which is not present on the backend. I think we should have that as `presence.Presence` and rename `PresenceStateBody` to `PresenceApiType` as that type represents what's coming from the backend API

Reply:
That makes sense! Thank you for the feedback!